src/internal/connector/exchange/service_functions.go

```diff
@@ -118,14 +121,18 @@ func includeContainer(
 		return nil, nil, false
 	}
 
-	locPath, err := pb.ToDataLayerExchangePathForCategory(
-		qp.Credentials.AzureTenantID,
-		qp.ResourceOwner,
-		category,
-		false)
-	// Containers without a path (e.g. Root mail folder) always err here.
-	if err != nil {
-		return nil, nil, false
+	var locPath path.Path
+
+	if loc != nil {
```

Comment:
we're still
1. comparing only against locPath
2. returning locPath

If we're only using locPath, then what's the point of having loc at all?